files/claude/skills/zod-extract/SKILL.md

```
@@ -0,0 +1,180 @@
+```
+
+## Exceptions
+
+Keep inline when:
+- One-off validation in a test file
+- Truly local validation that will never be reused
+- Extending/picking from an already-centralized schema in the same file
+
+## Output Format
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)